otel/metadata.csv

```diff
@@ -54,28 +54,92 @@ otelcol_processor_batch_batch_size_trigger_send,count,,unit,,Number of times the
-otel.process.runtime.go.mem.heap_alloc,gauge,15,byte,,Bytes of allocated heap objects,0,otel,otel runtime go mem heap alloc,
-otel.process.runtime.go.mem.heap_idle,gauge,15,byte,,Bytes in idle (unused) spans,0,otel,otel runtime go mem heap idle,
-otel.process.runtime.go.mem.heap_inuse,gauge,15,byte,,Bytes in in-use spans,0,otel,otel runtime go mem heap inuse,
-otel.process.runtime.go.mem.heap_objects,gauge,15,unit,,Number of allocated heap objects,0,otel,otel runtime go mem heap objects,
-otel.process.runtime.go.mem.heap_released,gauge,15,byte,,Bytes of idle spans whose physical memory has been returned to the OS,0,otel,otel runtime go mem heap released,
-otel.process.runtime.go.mem.heap_sys,gauge,15,byte,,Bytes of heap memory obtained from the OS,0,otel,otel runtime go mem heap sys,
-otel.process.runtime.go.mem.lookups,count,15,unit,,Number of pointer lookups performed by the runtime,0,otel,otel runtime go mem lookups,
-otel.process.runtime.go.mem.live_objects,gauge,15,unit,,Number of live objects is the number of cumulative Mallocs - Frees,0,otel,otel runtime go mem live objects,
-otel.process.runtime.go.gc.count,count,15,unit,,Number of completed garbage collection cycles,0,otel,otel runtime go gc count,
-otel.process.runtime.go.gc.pause_total_ns,count,15,nanosecond,,Cumulative nanoseconds in GC stop-the-world pauses since the program started,0,otel,otel runtime go gc pause total ns,
\ No newline at end of file
+runtime.uptime,count,10,millisecond,,Milliseconds since application was initialized,0,otel,runtime uptime,
+process.runtime.go.goroutines,gauge,10,invocation,,Number of goroutines that currently exist,0,otel,runtime go goroutines,
+process.runtime.go.cgo.calls,gauge,10,invocation,,Number of cgo calls made by the current process,0,otel,runtime go cgo calls,
+process.runtime.go.mem.heap_alloc,gauge,10,byte,,Bytes of allocated heap objects,0,otel,runtime go mem heap alloc,
+process.runtime.go.mem.heap_idle,gauge,10,byte,,Bytes in idle (unused) spans,0,otel,runtime go mem heap idle,
+process.runtime.go.mem.heap_inuse,gauge,10,byte,,Bytes in in-use spans,0,otel,runtime go mem heap inuse,
+process.runtime.go.mem.heap_objects,gauge,10,unit,,Number of allocated heap objects,0,otel,runtime go mem heap objects,
+process.runtime.go.mem.heap_released,gauge,10,byte,,Bytes of idle spans whose physical memory has been returned to the OS,0,otel,runtime go mem heap released,
+process.runtime.go.mem.heap_sys,gauge,10,byte,,Bytes of heap memory obtained from the OS,0,otel,runtime go mem heap sys,
+process.runtime.go.mem.lookups,count,10,unit,,Number of pointer lookups performed by the runtime,0,otel,runtime go mem lookups,
+process.runtime.go.mem.live_objects,gauge,10,unit,,Number of live objects is the number of cumulative Mallocs - Frees,0,otel,runtime go mem live objects,
+process.runtime.go.gc.count,count,10,unit,,Number of completed garbage collection cycles,0,otel,runtime go gc count,
```

Comment:
This one could be garbage collection as the unit maybe?

Reply:
This is also based off of the dd-trace-go metadata for runtime.go.mem_stats.num_gc: https://github.com/DataDog/dogweb/blob/prod/integration/go/go_metadata.csv#L29